client/service/service.go

```diff
@@ -0,0 +1,152 @@
+package boxservice
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"reflect"
+	runtimeDebug "runtime/debug"
+	"sync"
+	"time"
+	"unsafe"
+
+	box "github.com/sagernet/sing-box"
+	"github.com/sagernet/sing-box/adapter"
+	"github.com/sagernet/sing-box/common/urltest"
+	"github.com/sagernet/sing-box/experimental"
+	"github.com/sagernet/sing-box/experimental/clashapi"
+	"github.com/sagernet/sing-box/experimental/libbox"
+	"github.com/sagernet/sing-box/experimental/libbox/platform"
+	"github.com/sagernet/sing-box/option"
+	"github.com/sagernet/sing/service"
+	"github.com/sagernet/sing/service/pause"
+
+	"github.com/getlantern/radiance/client/boxoptions"
+	"github.com/getlantern/radiance/protocol"
+)
+
+// BoxService is a wrapper around libbox.BoxService
+type BoxService struct {
+	*libbox.BoxService
```

Comment:
gomobile doesn't support exporting pointers to types from external packages. That means it will skip this field and any methods from libbox (like `Start`) won't be accessible in the generated bindings. You can confirm this by running..

```bash
GOOS=ios gomobile bind -v  -target ios -trimpath -ldflags="-w -s" -o bin/LanternCore.xcframework ./client/service
cat bin/LanternCore.xcframework/ios-arm64_x86_64-simulator/LanternCore.framework/Headers/Boxservice.objc.h | grep "skipped field"

// skipped field BoxService.BoxService with unsupported type: *github.com/sagernet/sing-box/experimental/libbox.BoxService
```